src/app/models/form-solicitud.model.ts

```diff
@@ -0,0 +1,21 @@
+import { AbstractControl, ValidationErrors, ValidatorFn } from '@angular/forms';
+import { EdadDesdeFechaNacimientoPipe } from '../pipes/edad-desde-fecha-nacimiento.pipe';
+
+export function edadValida(): ValidatorFn {
+  return (control: AbstractControl): ValidationErrors | null => {
+    const rawValue = control.getRawValue();
+    const rawValueTime = Date.parse(rawValue);
+    const rawValueDate = new Date(rawValueTime);
+    
+    const edad: number = new EdadDesdeFechaNacimientoPipe().transform(rawValueDate);
+    if(edad > 17 && edad < 65){
```

Comment:
Pequeño detalle de performance en tiempo de ejecución: ahora mismo estarás instanciando una nueva pipe cada vez que el form control se valide. Se puede evitar eso si la pipe la instancias fuera de esta función, tal que así:

```suggestion
export function edadValida(): ValidatorFn {
 const edadDesdeFechaNacimientoPipe = new EdadDesdeFechaNacimientoPipe(); // la creas una única vez, al añadir el validador al formControl

  return (control: AbstractControl): ValidationErrors | null => {
    const rawValue = control.getRawValue();
    const rawValueTime = Date.parse(rawValue);
    const rawValueDate = new Date(rawValueTime);
    
    const edad: number = edadDesdeFechaNacimientoPipe .transform(rawValueDate); // se reutiliza N veces, tantas veces como se valide el formControl
    if(edad > 17 && edad < 65){
```